lib/api/index.ts

```diff
@@ -0,0 +1,50 @@
+type LPAResponse = {
+  entities: LPAEntity[];
+};
+
+type LPAEntity = {
```

Comment:
this is the rest of the data we are getting from PDP about the LPA entities. At the moment we are only using the entities names to populate the autocomplete but at some point we might want to look into using more of this data to populate the header (as pointed out by Eddie)